lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -592,4 +594,22 @@ bool SessionStateEnforcer::static_rule_init() {
   GlobalRuleList.insert_rule(8, reqpdr2);
   return true;
 }
+
+bool SessionStateEnforcer::set_upf_node(
+    const std::string& node_id, const std::string& addr) {
+  upf_node_id_      = node_id;
+  upf_node_ip_addr_ = addr;
+  MLOG(MDEBUG) << "Set_upf_node_id: " << upf_node_id_;
+  MLOG(MDEBUG) << "Set_upf_n3_addr: " << upf_node_ip_addr_;
+  return true;
+}
+
+std::string SessionStateEnforcer::get_upf_n3_addr() {

```

Comment:
`std::string SessionStateEnforcer::get_upf_n3_addr() const` might be good to mark them as const so it's clear it doesn't modify internal state.
Same for `get_upf_node_id`